src/ActivityFeeds/PostsFeedControls.jsx

```diff
@@ -228,6 +268,14 @@ if (!state.initialized && initialSelectedTab && initialSelectedTab !== state.sel
   selectTab(initialSelectedTab);
 }
 
+const optimisticPosts = Storage.get("optimisticPosts") ?? state.optimisticPosts ?? [];
+if (optimisticPosts.length > 0) {
+  const timestamp = optimisticPosts[0].block_timestamp;
+  if (!timestamp || Date.now() - timestamp / 1000000 > 60000) {
```

Comment:
This is an extra check in case the user never returned to the page after an error or cancel. After 1 minute, clear the optimistic update.